appveyor.yml

```diff
@@ -2,6 +2,7 @@ version: '{build}'
 branches:
   only:
   - master
+  - appveyor
```

Comment:
Yes - that's correct; there was just a weird webhook failure here.  But it encouraged me to add the `appveyor` branch so that we could build `appveyor` in isolation and not burn unnecessary travis builds.